core/src/main/java/com/oath/oak/Result.java

```diff
@@ -0,0 +1,23 @@
+package com.oath.oak;
+
+class Result<V> {
```

Comment:
class documentation please.
What does each boolean attribute represent?

Reply:
Will update documentation. This is basically a simple implementation of a sum type that can either hold a boolean value or a `V` value